js_modules/dagit/packages/core/src/runs/RunActionsMenu.tsx

```diff
@@ -58,8 +58,9 @@ import {useJobAvailabilityErrorForRun} from './useJobAvailabilityErrorForRun';
 
 export const RunActionsMenu: React.FC<{
   run: RunTableRunFragment;
+  additionalActions?: React.ReactNode[];
```

Comment:
This might be a little dicey for a memoized component -- will this be a memoized array?